build.sbt

```diff
@@ -7,8 +7,8 @@ homepage in ThisBuild              := Some(url("http://www.liftweb.net"))
 licenses in ThisBuild              += ("Apache License, Version 2.0", url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
 startYear in ThisBuild             := Some(2006)
 organizationName in ThisBuild      := "WorldWide Conferencing, LLC"
-scalaVersion in ThisBuild          := "2.12.6"
-crossScalaVersions in ThisBuild    := Seq("2.12.6", "2.11.11")
+scalaVersion in ThisBuild          := "2.12.8"
+crossScalaVersions in ThisBuild    := Seq("2.12.8", "2.11.11")
```

Comment:
Ditto re: 2.11.12.

Reply:
I'm happy to version bump that later but this has been languishing awhile, let's not hold it up.